Applications/ConsumerApi/test/ConsumerApi.Tests.Integration/StepDefinitions/RelationshipsStepDefinitions.cs

```diff
@@ -115,6 +137,16 @@ public async Task WhenAPostRequestIsSentToTheRevokeRelationshipEndpointByI2()
         _revokeRelationshipResponse = await _client1.Relationships.RevokeRelationship(_relationshipId, revokeRelationshipRequest);
     }
 
+    [When("a GET request is sent to the /Relationships/CanCreate\\?peer={i.id} endpoint by i1 for i2")]
+    public async Task WhenAGetRequestIsSentToTheCanCreateEndpointByI1()
+    {
+        _canEstablishResponse = await _client1.Relationships.CanCreateRelationship(_client2.IdentityData?.Address ?? "");
```

Comment:
I think in this case (since it's test code, and not production code) I would prefer the potential NullReferenceException over a silent fallback to an empty string.
```suggestion
        _canEstablishResponse = await _client1.Relationships.CanCreateRelationship(_client2.IdentityData!.Address);
```